java/java.hints/src/org/netbeans/modules/java/hints/perf/SizeEqualsZero.java

```diff
@@ -82,6 +93,23 @@ public static ErrorDescription sizeNotEqualsZero(HintContext ctx) {
         return sizeEqualsZeroHint(ctx, true);
     }
 
+    @TriggerPatterns({
+        @TriggerPattern(value="$subj.size() > 0",
+            constraints = @ConstraintVariableType(type = "java.util.Collection", variable = "$subj")),
+        @TriggerPattern(value="$subj.size() > 0",
+            constraints = @ConstraintVariableType(type = "java.util.Map", variable = "$subj")),
+        @TriggerPattern(value="0 < $subj.size()",
+            constraints = @ConstraintVariableType(type = "java.util.Collection", variable = "$subj")),
+        @TriggerPattern(value="0 < $subj.size()",
+            constraints = @ConstraintVariableType(type = "java.util.Map", variable = "$subj")),
+    })
+    public static ErrorDescription sizeGreaterZero(HintContext ctx) {
+        if (!ctx.getPreferences().getBoolean(CHECK_GREATER_ZERO, CHECK_NOT_EQUALS_DEFAULT)) {
+            return null;
+        }
+        return sizeEqualsZeroHint(ctx, true);
```

Comment:
is `not == true` correct in this invocation ?

Reply:
I believe so. Its essentially the same logic as the previous set of trigger patterns - another way of saying `sizeNotEqualsZero`.